src/System.ValueTuple/tests/System.ValueTuple.Tests.csproj

```diff
@@ -23,5 +25,11 @@
       <Project>{4C2655DB-BD9E-4C86-83A6-744ECDDBDF29}</Project>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="project.json" />
+  </ItemGroup>
```

Comment:
I'm not able to compile the tests with `TargetGroup=netcoreapp1.1` yet. Seems I'm missing some `XUnit` references. But I don't know how to get a suitable reference (the ones I tried pull in conflicting core types like Object).